### PR TITLE
Homogenize bound kwargs

### PR DESCRIPTION
**Pull request by stelmo**
`lower` -> `lower_bound` and `upper` -> `upper_bound` everywhere

### CODE REVIEW

File: src/reconstruction/MatrixCoupling.jl [hunk 1]
```diff
@@ -260,42 +260,58 @@ end
...
 end
 
 @_change_bounds_fn MatrixCoupling String inplace begin
-    change_bound!(model.lm, rxn_id, lower = lower, upper = upper)
+    change_bound!(model.lm, rxn_id, lower_bound = lower_bound, upper_bound = upper_bound)
 end
 
 @_change_bounds_fn MatrixCoupling String inplace plural begin
-    change_bounds!(model.lm, rxn_ids, lower = lower, upper = upper)
+    change_bounds!(model.lm, rxn_ids, lower_bound = lower_bound, upper_bound = upper_bound)
```
Comment (exaexa) [inline src/reconstruction/MatrixCoupling.jl:275]: these can be done simpler using `;` in the right place:
```suggestion
    change_bounds!(model.lm, rxn_ids; lower_bound, upper_bound)
```

Comment (exaexa) [reply src/reconstruction/MatrixCoupling.jl:275]: (same in many places around in the file)

File: src/reconstruction/MatrixModel.jl [hunk 2]
```diff
@@ -254,52 +254,62 @@ function verify_consistency(
...
+        upper_bound = [upper_bound],
+    )
 end
 
 @_change_bounds_fn MatrixModel Int plural begin
     n = copy(model)
     n.xl = copy(n.xl)
     n.xu = copy(n.xu)
-    change_bounds!(n, rxn_idxs, lower = lower, upper = upper)
+    change_bounds!(n, rxn_idxs, lower_bound = lower_bound, upper_bound = upper_bound)
```
Comment (exaexa) [inline src/reconstruction/MatrixModel.jl:281]: ..and here

File: src/reconstruction/MatrixModel.jl [hunk 3]
```diff
@@ -254,52 +254,62 @@ function verify_consistency(
...
 end
 
 @_change_bounds_fn MatrixModel String inplace plural begin
     change_bounds!(
         model,
         Vector{Int}(indexin(rxn_ids, variables(model))),
-        lower = lower,
-        upper = upper,
+        lower_bound = lower_bound,
+        upper_bound = upper_bound,
```
Comment (exaexa) [inline src/reconstruction/MatrixModel.jl:299]: and here

File: src/reconstruction/ObjectModel.jl [hunk 4]
```diff
@@ -193,7 +193,7 @@ end
     for i in rxn_ids
         n.reactions[i] = copy(n.reactions[i])
     end
-    change_bounds!(n, rxn_ids, lower = lower, upper = upper)
+    change_bounds!(n, rxn_ids, lower_bound = lower_bound, upper_bound = upper_bound)
```
Comment (exaexa) [inline src/reconstruction/ObjectModel.jl:196]: and here

**Review by exaexa — APPROVED** [review]
LGTM FFTM

I added a few comment about keyword arguments, might be nice to simplify the code that way. It's a complete nitpick tho.

File: src/reconstruction/MatrixCoupling.jl (hunk 1 above)
Comment (stelmo) [reply src/reconstruction/MatrixCoupling.jl:275]: will fix all these

**Pull request merged**